### PR TITLE
Update to simplified build.

### PR DESCRIPTION
**Pull request by adevnadia**
Update to simplified build. Update build artifacts and simplify dev process.

### CODE REVIEW

File: test/unit/slot/polyfill.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import '../../../src/index';
-import create from '../lib/create';
+import create from '../../lib/create';
```
Comment (jpnelson) [inline test/unit/slot/polyfill.js:2]: is this related to a build change?

**Comment by treshugart** [comment]
lgtm

File: test/unit/slot/polyfill.js (hunk 1 above)
Comment (adevnadia) [inline test/unit/slot/polyfill.js:2]: Not exactly, it was broken for some time, don't know how did it worked until now.
When I changed to the new build system it's just revealed.
